assembly/std/index.ts

```diff
@@ -139,6 +139,16 @@ export function getOpKeys(): Array<StaticArray<u8>> {
   return derOpKeys(keysSer);
 }
 
+/**
+ * Get all keys from datastore
+ *
+ * @return {StaticArray<u8>} - a list of key (e.g. a list of bytearray)
+ */
+export function getKeys(address: string): Array<StaticArray<u8>> {
+  let keysSer = env.getKeys(address);
+  return derOpKeys(keysSer);
```

Comment:
Why to you use `derOpKeys` for this also? This function works in this case also I assume. Then I think you should rename it to show that it works on multiple usage.

Reply:
Renamed